### DISCUSSION TITLE
subsections

### DISCUSSION THREAD

**Discussion by Qfrijters** [nextcloud/cookbook#1830] [Ideas and discussions]
I am currently  typing in a recipe, and it has subsections for ingredients and I was wondering if it would be possible to be able to have a whitespace or a colour.
This might sound vague, but maybe this example will help

They way it is done now, is like this
![msedge_70p9EGYnuN](https://github.com/nextcloud/cookbook/assets/68586456/04b77dcd-3984-45ba-ab9a-3396c07df0f7)

I was wondering if it was possible to do it like this

Bottom:
-  150g butter
-  170 g sugar
-  180g flower

filling:
- insert ingredients
- more ingredients
- even more here

toplayer: 
- ingredients
- ingredient
- ingredient

Thanks :)

**Comment by christianlupus** [comment]
You might want to add the headings as normal ingredients with `## ` before. This should create a grouping if I am right.

**Reply by Qfrijters** [reply]
Oh that works
Cheers
